grails-app/views/order/_orderItemFormDialog.gsp

```diff
@@ -0,0 +1,216 @@
+<%@ page import="org.pih.warehouse.order.OrderStatus" %>
+<g:form name="editOrderItemForm" action="purchaseOrder" method="post">
+    <g:hiddenField id="dlgOrderId" name="order.id" value="${orderItem?.order?.id}" />
+    <g:hiddenField id ="dlgOrderItemId" name="orderItem.id" value="${orderItem?.id}"/>
+    <table>
+        <tbody>
+
+        <g:if test="${orderItem?.order.status <= org.pih.warehouse.order.OrderStatus.PENDING}">
+            <tr class='prop'>
```

Comment:
You were using double quotes in previous file and in this majority are single quotes. I suggest trying sticking to one pattern

Reply:
Agreed. This is an old code fragment that I moved from the showOrderItems.gsp to its own template to make the former easier to read.